react-bootstrap/index.d.ts

```diff
@@ -590,6 +590,7 @@ declare namespace ReactBootstrap {
         active?: boolean;
         noCaret?: boolean;
         eventKey?: any;
+        title?: any; // TODO: Add more specific type
     }
     type NavDropdownProps = NavDropdownBaseProps & React.HTMLProps<NavDropdown>;
```

Comment:
@vsiao: So it should be union? Something like `title?: ReactNode|string`? Or does it need separate type like this?